docs/extend/modular-resources/configure.md

```diff
@@ -193,46 +194,89 @@ The custom model is configured as a component with the name "my-realsense".
+   - If you are adding a modular [component](/components/), click the **Components** subtab and click **Create component**.
+   - If you are adding a modular [service](/services/), click the **Services** subtab and click **Create service**.
+
+1. Then, select the `local modular resource` type from the list.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-list.png" resize="300x" declaredimensions=true alt="The add a component modal showing the list of components to add with 'local modular resource' shown at the bottom">}}
+
+1. On the next screen:
+   - Select the type of modular resource provided by your module, such as a [camera](/components/camera/), from the drop down menu.
+   - Enter the {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} of your modular resource's [model](/extend/modular-resources/key-concepts/#models).
+     If you are adding a pre-built modular resource, the model triplet should be provided for you in the module's documentation.
+   - Enter a name for this instance of your modular resource.
+     This name must be different from the module name.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-create.png" resize="400x" declaredimensions=true alt="The add a component modal showing the create a module step for an intel realsense module">}}
+
+1. Click **Create** to create the modular resource provided by the local module.
 
-### Configure your module
+You can also add the module directly, without first adding its modular component or service:
 
-To configure your new *module* on your robot, navigate to the **Config** tab of your robot's page on [the Viam app](https://app.viam.com) and click on the **Modules** subtab.
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Modules** subtab.
+1. Scroll to the **Add local module** section.
+1. Enter a **Name** for this instance of your modular resource.
+1. Enter the [module's executable path](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+   This path must be the absolute path on the filesystem to the executable.
+1. Then, click the **Add module** button, and click **Save config**.
+
+   {{<imgproc src="extend/modular-resources/configure/add-local-module-csi-cam.png" resize="600x" declaredimensions=true alt="The add a local module pane with name 'my-csi-ca' and executable path '/usr/local/bin/viam-csi'">}}
+
+   This example shows the configuration for adding a [CSI camera](/extend/modular-resources/examples/csi/) as a local module.
+
+## Configure a local module
+
+Once you have added a modular resource to your robot, you can view and configure the module itself from the **Modules** subtab:
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Modules** subtab.
```

Comment:
```suggestion
2. Click on the **Modules** subtab.
```

Reply:
MD will fix this anyways

Reply:
This one's convention. MD actually completely ignores the numbers you choose, so using 1. for all is a common convention. But it could easily be matched to desired HTML output as well, that's just a little more cumbersome to later interpolate additional items, in following PRs. With `1.` throughout, a new item or existing item can be added or edited without touching any others.